pkg/test/framework/components/environment/kube/flags.go

```diff
@@ -201,7 +201,10 @@ func init() {
 	flag.StringVar(&kubeConfigs, "istio.test.kube.config", strings.Join(settingsFromCommandLine.KubeConfig, ":"),
 		"A comma-separated list of paths to kube config files for cluster environments (default is current kube context)")
 	flag.BoolVar(&settingsFromCommandLine.Minikube, "istio.test.kube.minikube", settingsFromCommandLine.Minikube,
-		"Indicates that the target environment is Minikube. Used by Ingress component to obtain the right IP address..")
+		"Indicates that the target environment is Minikube. Used by to obtain the right IP address for ingress gateway.")
```

Comment:
I assume that when running with kind, both will be true? AFAIK the `minikube` flag probably should have been renamed `noexternalloadbalancer` (or something similar)... it was never really about minikube, itself. Would it make more sense to just run with minikube=false when our environment uses metallb?

Reply:
Agreed. I'm pretty sure all references to the minikube flag are self-contained to this repo and its wiki. I'll probably keep the minikube flag for backwards-compat.